ocaml/xapi/dbsync_master.ml

```diff
@@ -89,7 +89,7 @@ let refresh_console_urls ~__context =
                 | Some x when x = Unix.PF_INET ->
                   address
                 | Some x when x = Unix.PF_INET6 ->
-                  "[" ^ address ^ "]"
+                  Http.Url.maybe_wrap_IPv6_literal address
```

Comment:
Does this fix a bug where we might have accidentally wrapped addresses in two sets of braces `[]`?

Reply:
If a port si specified then the `[]` are already present.
I don't know if it can happen in this specific case but I think using a dedicated method is better than handling manually.

Reply:
Indeed - a good spot :+1: